Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -1395,10 +1387,6 @@
       <Project>{1ae2cb5c-58a0-4f12-8e6f-2cd4aaadb34c}</Project>
       <Name>Microsoft.Toolkit.Uwp.Samples.BackgroundTasks</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Microsoft.Toolkit.Uwp.Services\Microsoft.Toolkit.Uwp.Services.csproj">
```

Comment:
Think this is still used by a couple of the other services in the sample app.  Maybe we should split this part out of this change and keep it scoped to the ARM64 work-around?

Also, were we going to remove the pfx?

Reply:
They are all deprecated now, so all I did was remove the only code reference that it still had, which is deprecated. We still link to the markdown, so docs are still there.
I've updated the PFX, but we can remove it later. Do you think we need it to be be removed on this PR?